app/src/androidTest/java/org/mozilla/fenix/ui/MediaNotificationTest.kt

```diff
@@ -161,5 +165,6 @@ class MediaNotificationTest {
 
         // close notification shade before the next test
         mDevice.pressBack()
+        homeScreen { }.togglePrivateBrowsingMode()
     }
```

Comment:
I would add a comment why this is needed.  This looks fine to me if this ensures the notification shade is closed.

Reply:
yes, I have seen in the failures video that the next test starts on private mode while it should not, trying to reduce  the failure if we start in regular mode as expected